devfiles/java-mongo/meta.yaml

```diff
@@ -0,0 +1,6 @@
+---
+displayName: Java with Spring Boot and MongoDB
+description: Java stack with OpenJDK 8, MongoDB and Spring Boot Guestbook demo application
+tags: ["Java", "OpenJDK", "Maven", "Spring Boot", "MongoDB"]
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+globalMemoryLimit: 3284Mi
```

Comment:
A memory limit of `3284Mi` will prevent this devfile from running on che.openshift.io. Is there any way to reduce the required memory below `3Gi`?

Reply:
Spring boot pulls in a boatload of maven dependencies. Is using it a choice or are there good reasons for it?

Reply:
@amisevsk I set 300Mi for mongo container and 512 for maven, tested it and looks like it works without problems. So, now global memory limit is 2884Mi.

Reply:
:+1: